hedera-node/hedera-admin-service/src/main/java/com/hedera/node/app/service/admin/FreezePreTransactionHandler.java

```diff
@@ -34,5 +35,5 @@ public interface FreezePreTransactionHandler extends PreTransactionHandler {
      *     com.hederahashgraph.api.proto.java.FreezeTransactionBody}
      * @return the metadata for the freeze
      */
-    TransactionMetadata preHandleFreeze(TransactionBody txn);
+    TransactionMetadata preHandleFreeze(TransactionBody txn, AccountID payer);
```

Comment:
Will change to use `SigTransactionMetadataBuilder`  once `createStore` method is added, since `SigTransactionMetadataBuilder` needs `AccountKeyLookup` passed in from `PreHandleWorkFlow` @netopyr

Reply:
yeah true !